### DISCUSSION TITLE
GitHub Mobile can't show merge events for PRs that were previously in the merge queue normally

### DISCUSSION THREAD

**Discussion by kaikaibenkai** [community/community#112910] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

This is github-linguist/linguist#6749 on github.com

![Screenshot_20240317_130309_com microsoft emmx](https://github.com/community/community/assets/99722991/78be2d45-4ae8-4065-be81-6eea50a8be62)

But in GitHub Mobile, it seems a bit strange (closing again after merging?)
![Screenshot_20240317_130459_com github android](https://github.com/community/community/assets/99722991/6b31c8a4-05be-435c-ab95-bfcb7f0497f1)

**Comment by esahmed21** [comment, marked as answer]
Thank you, @kaikaibenkai, for bringing this to our attention! I was able to see a similar issue on iOS, where the timeline displays closed event after merged event( bur with different icon colors). It happens for this specific mentioned pull request, I'll create an internal ticket for our team to investigate and address this issue..
